app/src/main/java/ch/hikemate/app/model/facilities/FacilitiesRepositoryOverpass.kt

```diff
@@ -52,12 +52,14 @@ class FacilitiesRepositoryOverpass(private val client: OkHttpClient) :
     override fun onResponse(call: okhttp3.Call, response: okhttp3.Response) {
       try {
         if (!response.isSuccessful) {
-          onFailure(
-              Exception(
-                  "Failed to fetch facilities from Overpass API. Response code: ${response.code}"))
+          onFailure(Exception())
```

Comment:
Question: Why did you have to remove the exception strings? They are useful for debugging and logging, no?

Reply:
It was just to make sure we never use them to display anything in the UI. It's more of a prevention than a necessary remove.
I agree that could help for debugging. I will put them back.